daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -94,6 +74,16 @@ object Value extends CidContainer1[Value] {
     go
   }
 
+  def cids[Cid2 >: ContractId] = {
+    val cids = Set.newBuilder[Cid2]
+    foreach1(cids += _)
```

Comment:
Hi @cocreature , it does not look like `foreach1(cids += _)` adds any contract ids to the builder as `Value.foreach1` with a single parameter returns another function.

Caught by a couple of innocent-looking canton unit tests.

Reply:
Proposed fix in https://github.com/digital-asset/daml/pull/11032